src/css/components/table.css

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file
+ * Styles for Olivero's tables.
+ */
+ 
+table {
+  width: fit-content;
+  max-width: 100%;
+  border-collapse: collapse;
+  display: block;
+  margin: var(--sp2) 0;
+  border-spacing: 0;
+  border: 0;
+  font-family: var(--font-sans);
+  line-height: var(--sp1-5);
+  font-size: 16px;
+  color: var(--color--gray-10);
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+
+  /* Hide scrollbar */
+  scrollbar-width: none; /* Firefox */
```

Comment:
I think we should not change the browser's default behavior here. It's probably an a11y issue.

Reply:
Yep! You're probably right.